### PR TITLE
Updaten npmrc file 

### PR DESCRIPTION
**Pull request by alfredo-pozo**
Switch to public npm registry

Change of email address notification

Update of docker image that will contain the file with the private npm registry

This PR is related to this task https://jira.autodesk.com/browse/DYN-3200

### CODE REVIEW

File: .npmrc [hunk 1]
```diff
@@ -1 +1 @@
-registry=https://art-bobcat.autodesk.com/artifactory/api/npm/autodesk-npm-virtual/
\ No newline at end of file
+registry=https://registry.npmjs.org
```
Comment (mjkkirschner) [inline .npmrc:1]: so is there another .npmrc file for the build that points to art-bobcat?

File: pipeline.yml [hunk 2]
```diff
@@ -22,8 +22,9 @@ build:
   -
     customized_builder_image:
       registry: artifactory
-      url: artifactory.dev.adskengineer.net/docker-local-v2/dynamo/dynamolibrariejs:dotnetcore2
+      url: artifactory.dev.adskengineer.net/docker-local-v2/dynamo/dynamolibrariejs:dotnetcore3
```
Comment (mjkkirschner) [inline pipeline.yml:25]: has this docker file/docs added to the dynamo build resources repo yet?

File: .npmrc (hunk 1 above)
Comment (alfredo-pozo) [reply .npmrc:1]: That file will be switched during CI/CD process as requested on the Jira task

File: pipeline.yml (hunk 2 above)
Comment (alfredo-pozo) [reply pipeline.yml:25]: Yes, feel free to check the last PR that was merged today

Comment (mjkkirschner) [reply pipeline.yml:25]: thanks!

**Review by QilongTang — APPROVED** [review]

**Review by QilongTang — APPROVED** [review]

**Pull request merged**